src/Qt-RSA/qrsaencryption.cpp

```diff
@@ -290,20 +289,76 @@ QByteArray decodeArray(const QByteArray &rawData, const QByteArray &privKey) {
     return res.remove(res.lastIndexOf(ENDLINE), res.size());
 }
 
-bool QRSAEncryption::testKeyPair(const QByteArray &pubKey, const QByteArray &privKey) {
-    QByteArray tesVal = "Test message of encrypkey";
+QRSAEncryption::QRSAEncryption(Rsa _keyLength)
```

Comment:
We do not need a constructor with RSAbit.
RSAbit rate is passed to the 3rd parameter in the methods of encryption and key generation